README.md

```diff
@@ -238,28 +259,31 @@ describe(‘MyComponent’, function() {
 
 ### Performance Considerations
 
- * Connections in separate modules, load what you need.
- * Abortable requests.
- * Optionally cold connections.
+ * Connections in separate modules, load what you need
+ * Abortable requests
+ * Optionally cold connections
```

Comment:
it might be good to go into more details on this, the hot/cold nomenclature was new to me in this context